gocourse04/zoo/sector_test.go

```diff
@@ -0,0 +1,37 @@
+package zoo
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewSector(t *testing.T) {
+	sector := NewSector("Test sector")
+
+	assert.Equal(t, "Test sector", sector.Subtype)
+	assert.Equal(t, 0, len(sector.Animals))
+}
+
+func TestSectorAddAnimal(t *testing.T) {
+	sector := *NewSector("Test sector")
+	animal := NewAnimal(AllAnimalSpeciesValues()[0], 5, "qwe")
+
+	sector = sector.AddAnimal(animal)
+
+	assert.Equal(t, 1, len(sector.Animals))
+	assert.Equal(t, 5, sector.Animals[0].ID)
+}
+
+func TestSectorRemoveAnimal(t *testing.T) {
+	sector := *NewSector("Test sector")
```

Comment:
Спробуємо обійтись без зірочок:
```suggestion
	sector := NewSector("Test sector")
```